### PR TITLE
Rename fixtures paths to be consistent 

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no (maybe)
| New feature?  | no
| Deprecations? | no
| License       | MIT

There are currently two components with a "fixtures" (lowercase) folder,
* [Symfony/Component/AssetMapper/Tests/fixtures](https://github.com/symfony/symfony/tree/6.4/src/Symfony/Component/AssetMapper/Tests/fixtures)
* [Symfony/Component/Translation/Tests/fixtures](https://github.com/symfony/symfony/tree/6.4/src/Symfony/Component/Translation/Tests/fixtures)

So on case-sensitive filesystems the following exclusion in php-cs-fixer does not work.

```php
(new PhpCsFixer\Finder())
    ->in(__DIR__.'/src')
    ->append([__FILE__])
    ->notPath('#/Fixtures/#')
```    

Those are the files who are currently not excluded on case-sensitive filesystems

```console
php-cs-fixer list-files | grep fixtures
'./src/Symfony/Component/Translation/Tests/fixtures/extractor-ast/translation.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor-ast/translatable-short.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor-ast/translatable-fqn.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor-ast/translatable.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor-ast/validator-constraints.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor/translation.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor/translatable-short.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor/translatable-fqn.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor/translatable.html.php'
'./src/Symfony/Component/Translation/Tests/fixtures/extractor-7.3/translation.html.php'
'./src/Symfony/Component/AssetMapper/Tests/fixtures/importmap.php'
'./src/Symfony/Component/AssetMapper/Tests/fixtures/importmaps/importmap.php'
'./src/Symfony/Component/AssetMapper/Tests/fixtures/AssetMapperTestAppKernel.php'
```

This PR exclude files in both senstivie / insenstive systems.

### CODE REVIEW

**Comment by OskarStark** [comment]
What about renaming those folders?

**Comment by smnandre** [comment]
> What about renaming those folders?

I didn't dare to suggest. Really.

**Comment by smnandre** [comment]
@OskarStark there is the "rename" option. What do you think ?

(if this pass, all in-progress PR will probably hate me)

fabbot fails because there is an invalid json file (the same there was but the path has changed...)

**Comment by OskarStark** [comment]
Don't worry, they can rebase their PRs

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @smnandre.

File: src/Symfony/Component/Translation/Tests/Loader/PoFileLoaderTest.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function testSkipFuzzyTranslations()
     public function testMissingPlurals()
     {
         $loader = new PoFileLoader();
-        $resource = __DIR__.'/../fixtures/missing-plurals.po';
+        $resource = __DIR__ . '/../Fixtures/missing-plurals.po';
```
Comment (keradus) [inline src/Symfony/Component/Translation/Tests/Loader/PoFileLoaderTest.php:113]: this is against the Fixer config and caused lot of extra output to PHP CS Fixer results.
let's fix it.

Comment (keradus) [reply src/Symfony/Component/Translation/Tests/Loader/PoFileLoaderTest.php:113]: https://github.com/symfony/symfony/pull/52317

**Pull request merged**